generators/app/templates/package-scripts.js

```diff
@@ -1,4 +1,4 @@
-const npsUtils = require('nps-utils')
+const npsUtils = require('nps-utils') // eslint-disable-line
```

Comment:
sorry i mean that line clashes with 2 eslint rules.
eslint suggests it to have new line, and also suggests npsUtils to be listed as dependencies